docs/src/main/sphinx/release/release-367.md

```diff
@@ -0,0 +1,34 @@
+# Release 367 (XX Dec 2021)
+
+## General
+
+* Improve retryable query performance by adding support for dynamic filtering. ({issue}`10274`)
+* Improve `CREATE VIEW` error message when table or materialized view already exists. ({issue}`10186`)
+* Fix failure in `min_by` and `max_by` aggregation execution. ({issue}`10347`)
+* Fix planning failure when selecting all fields from a row type. ({issue}`10321`)
```

Comment:
IIUC the issue correctly it was not only regarding `ROW(..).*` but also if subscript on row was used in query directly `ROW(...)[1]`.

Then maybe such phrasing would be more accurate:
```
Fix failure of specific queries which accessed fields of `row` column using index based subscript operator (`ROW(...)[n]`) or selecting all row fields (`ROW(..).*`)
```

Reply:
Yes, it is. I was trying to describe the general area that might be affected without making it too convoluted. I'll update with a variant based on your wording.